dynatrace_health/README.md

```diff
@@ -0,0 +1,41 @@
+# Agent Check: Dynatrace_health
+
+## Overview
+
+This check monitors [Dynatrace_health][1] through the StackState Agent.
+
+## Setup
+
+### Installation
+
+The Dynatrace_health check is included in the [StackState Agent][2] package, so you do not
+need to install anything else on your server.
+
+### Configuration
+
+1. Edit the `dynatrace_health.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your dynatrace_health performance data.
+   See the [sample dynatrace_health.d/conf.yaml][2] for all available configuration options.
+
+2. Restart the Agent
+
+## Data Collected
```

Comment:
I don't see what does this check collects because it mentioned nothing is collected like topology, metrics , events and service checks.

Reply:
Fixed the Data colleceted section